database/devops/kubernetes.json

```
@@ -5,5 +5,12 @@
         "url": "https://kubernetes.io/docs/tutorials/kubernetes-basics/",
         "category": "devops",
         "subcategory": "kubernetes"
-    }
+    },
+{
+        "name": "Kubernetes Tutorial for Beginners [FULL COURSE in 4 Hours].",
+        "description": "This is a comprehensive Kubernetes tutorial for beginners.",
+        "url": "hhttps://youtu.be/X48VuDVv0do?si=dKDO8pGI9Wge3lN9",
+        "category": "youtube",
```

```suggestion
        "category": "devops",
```

Try to follow the guidelines, and check it locally. If you don't even see the preview, then how will you work on good issues.

It would never run locally. No worries, just fix it.